build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -25,11 +45,28 @@ data:
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
     $env:PATH = $env:PATH + ";$mountPath/Windows/System32;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
+    $OVSDriverDir="$mountPath\openvswitch\driver"
+    $WorkDir="$mountPath\openvswitch\"
+
+    # Check if OVSExt driver is already installed
+    $driverStatus = netcfg -q ovsext                                                                           
+    if ($driverStatus -like '*not installed*') { 
+    # Install OVS Driver
+    net start vmms
+    cd $mountPath/openvswitch/driver
+    netcfg -l .\ovsext.inf -c s -i OVSExt
+    cd $mountPath/openvswitch 
+    }
+
+    #Configure OVS processes
     $OVS_DB_SCHEMA_PATH = "$mountPath/openvswitch/usr/share/openvswitch/vswitch.ovsschema"
     $OVS_DB_PATH = "C:\openvswitch\etc\openvswitch\conf.db"
     if ($(Test-Path $OVS_DB_SCHEMA_PATH) -and !$(Test-Path $OVS_DB_PATH)) {
+        mkdir -force C:\openvswitch\etc\openvswitch\
         ovsdb-tool create "$OVS_DB_PATH" "$OVS_DB_SCHEMA_PATH"
     }
+
+    mkdir -force C:/openvswitch/var/run/openvswitch
```

Comment:
Question: would it overwrite the directory (`C:/openvswitch/var/run/openvswitch`) in restart case ?
It doesn't look good to create the OVS run path every time when initiating antrea-agent Pod.